src/main/java/com/exasol/projectkeeper/PomFileTemplateRunner.java

```diff
@@ -26,7 +26,10 @@
  */
 public class PomFileTemplateRunner {
 
-    public static final Collection<PomTemplate> TEMPLATES = List.of(new VersionMavenPluginPomTemplate());
+    public static final Collection<PomTemplate> TEMPLATES = List.of(new VersionMavenPluginPomTemplate(),
```

Comment:
And I think it's not a pom template, it's a plugin template, isn't it?

Reply:
It's a template that validates some aspect of the pom file. The ones that exist until now validate plugins, but there are also other possible.